setup.cfg

```diff
@@ -27,21 +28,23 @@ install_requires =
     nipype ~= 1.4
     nitime
     nitransforms >= 20.0.0rc3,<20.2
-    niworkflows @ git+https://github.com/nipreps/niworkflows.git@master
+    niworkflows @ git+https://github.com/oesteban/niworkflows.git@enh/335-pybids-datasinks
     numpy
     pandas
     psutil >= 5.4
     pybids >= 0.10.2
     pyyaml
-    sdcflows ~= 1.2.3
-    smriprep @ git+https://github.com/poldracklab/smriprep.git@master
+    sdcflows @ git+https://github.com/oesteban/sdcflows.git@enh/new-datasinks
+    smriprep @ git+https://github.com/oesteban/smriprep.git@enh/new-datasinks
     tedana >= 0.0.5
     templateflow >= 0.5.2
     toml
 test_requires =
     coverage
     codecov
     pytest
+setup_requires =
+    setuptools >= 40.8.0
```

Comment:
I've kept 40.8.0 because it was already the pinned version.

While struggling with some sdist building issues (I finally discovered that ``setuptools-scm``, now a dependency to sdcflows, basically defenestrates the ``MANIFEST.in`` and auto-includes anything tracked by git), incidentally I learned the key versions of setuptools to work with `setup.cfg`:

```
setup_requires =
    # setup.cfg supported by +30.3.0
    setuptools >= 38.3.0  # +38.3 contains most of bugfixes
```
Noted here: https://github.com/nipreps/niworkflows/pull/507/files#diff-380c6a8ebbbce17d55d50ef17d3cf906R45-R47

This is not something I really care much (as long as we make sure `setuptools >= 38.3.0`).